### ISSUE TITLE
Windows key blocked on Windows server

### ISSUE THREAD

**Issue by Vizzard** [symless/synergy#4486]
...  after locking with Win+L with mouse on Ubuntu client.

Synergy 1.6.3 x64 on both Windows (server) and Ubuntu 14.10 (client).

I found out that it is possible to lock both screens with Win+L having cursor on client (Ubuntu) side. After this, mouse goes back to center of the server (Windows) screen, probably due to this issue: https://github.com/synergy/synergy/issues/4485

Windows key remains blocked in "pressed" position. If you have password with "u" letter then on lock screen "Ease of Access" window will appear (Win+U combination). That's how I learned Win button is still pressed.

After pressing Windows button it gets released and everything is back to normal.

**Comment by curlessb** [comment]
Synergy 1.6.3 on two Windows 7 x64 workstations. 

I am unable to lock the client workstation with Win+L, it simply locks the server workstation, (when the cursor is on the client). I will agree with Vizzard that when this happens the mouse moves back to the center of the server immediately when I attempt to lock the client machine. Thereafter, once connection is restored, (before unlocking the server), if I move the cursor back and forth between the machines, the cursor always jumps to the middle of the (locked) server screen.

**Comment by cleung2010** [comment]
I also have this issue on Windows + Mac. The Win button remains pressed so any mouse clicks I do is a click + Cmd (since Win button maps to Cmd on the Mac side).

**Comment by the-wes** [comment]
@curlessb - Win+L doesn't work on your client because Windows won't accept it being generated by a non-Elevated process. If your Synergy were elevated, it would probably work as @Vizzard described.

**Comment by dwink** [comment]
I also have this issue -- I use a Linux environment that uses Win-L as a shortcut, and with a Synergy server on Windows, hitting that key-combination locks the windows desktop and -- more annoyingly -- moves the mouse cursor to the center of the now-locked desktop. It takes a few tries to get the mouse to move back to the Linux side -- it looks like it's kicking the client out and then taking a few seconds to allow a reconnect.

**Comment by XinyuHou** [comment]
Duplicate of #5574

**Comment by sethdmoore** [comment]
So both #5574 and #4486 have been closed? Sorry, but this isn't fixed yet.
